x-pack/dev-tools/jest/create_jest_config.js

```diff
@@ -46,6 +46,7 @@ export function createJestConfig({
     ],
     transform: {
       '^.+\\.(js|tsx?)$': `${kibanaDirectory}/src/dev/jest/babel_transform.js`,
+      '^.+\\.html?$': 'jest-raw-loader',
```

Comment:
is it related to the PR? don't any files in `x-pack` folder touched

Reply:
Because core now imports `src/legacy/ui/public/error_auto_create_index/error_auto_create_index.html` there are some x-pack jest tests which started loading this file, even though it's completely unrelated to the tests.